src/goto-programs/instrument_preconditions.cpp

```diff
@@ -146,3 +146,9 @@ void instrument_preconditions(goto_modelt &goto_model)
   for(auto &f_it : goto_model.goto_functions.function_map)
     remove_preconditions(f_it.second.body);
 }
+
+void remove_preconditions(goto_modelt &goto_model)
+{
+  for(auto &f_it : goto_model.goto_functions.function_map)
+    remove_preconditions(f_it.second.body);
```

Comment:
Expose the per-function version(s) too? Useful for `process_goto_function`, for example

Reply:
done